### DISCUSSION TITLE
Nueva instalación de Globaleaks

### DISCUSSION THREAD

**Discussion by rpg2023** [globaleaks/globaleaks-whistleblowing-software#3576] [Q&A]
Buenas tardes, 
acabo de desplegar Globaleaks en un servidor Debian 11.
Tengo acceso a la configuración y acceso a la plataforma desde la intranet mediante localhost http://127.0.0.1:8082 pero no puedo acceder a ella desde Internet. 
He configurado una IP publica y activado los puertos 80 y 443 en el contrafuegos, pero en lugar de salir la plataforma me sale la pagina por defecto de Apache diciendo que el servicio está activo y que configura mi propia página. 
Preguntas:
1.- mediante el asistente de instalación se puede configurar el servicio https en una IP pública o tengo que configurar manualmente los ficheros de configuración el servicio apache2 ?
2.- mediante el asistente de instalación se puede instalar un certificado de servidor seguro o tengo que instalarlo manualmente mediante consola de comandos.

Muchas gracias por su tiempo.
Un saludo.

**Comment by evilaliv3** [comment]
Hola!

> Message ID: ***@***.***>
>

Actually you do not need to have apache or any other application proxy.

Just install GlobaLeaks as it includes all that is needed.

After installing it you will be able to setup HTTPs via the web interface

**Reply by rpg2023** [reply]
Perfecto, creo que el demonio APACHE2 esta interfiriendo con el servicio globaleaks, ha sido pararlo, reiniciar el servicio y funcionar todo.
Otra duda, estoy intentado configurar el certificado para tener una navegación segura, pero me da un error en el código de país y no me deja generarlo. Adjunto captura de pantalla con el error.

Muchas gracias por el rápido y eficiente soporte.
Un saludo.
![errorClavePais](https://github.com/globaleaks/GlobaLeaks/assets/141417925/6dd82fe7-1603-43b8-81e7-9090988f7d78)

**Comment by evilaliv3** [comment]
The country code should be ES

**Comment by rpg2023** [comment]
Perfecto, con el código "ES" ya me ha generado el fichero "csr.pem"

Qué necesito hacer ahora para activar el certificado y que la navegación sea segura en el portal canaldenuncias.dipucuenca.es

Gracias.
Un saludo.


Rafael Pérez García

Técnico de Innovación, Modernización y Administración Electrónica

Diputación Provincial de Cuenca

969-177-197

________________________________
De: Giovanni Pellerano ***@***.***>
Enviado: viernes, 11 de agosto de 2023 14:19
Para: globaleaks/GlobaLeaks ***@***.***>
Cc: Rafael Pérez García ***@***.***>; Author ***@***.***>
Asunto: Re: [globaleaks/GlobaLeaks] Nueva instalación de Globaleaks (Discussion #3576)


The country code should be ES

**Comment by rpg2023** [comment]
Buenos días, 
necesito navegación segura nuestro portal http://canaldenuncias.dipucuenca.es/
Como lo configuro? Adjunto captura de pantalla
Gracias.
Un saludo.

![https](https://github.com/globaleaks/GlobaLeaks/assets/141417925/08453a9c-36be-4056-a40b-bf1a436e740d)

**Reply by Marcbcn23** [reply]
Hola rpg2023, me encuentro en las mismas. Como lo conseguiste finalmente? No consigo que mi servidor le asigne una dirección que no incluya el puerto de entrada y no consigo ese dichoso certificado. Como fue finalmente? Gracias!

**Comment by rpg2023** [comment]
Buenos días,
lamentablemente tenía que poner el servicio en producción y no podía seguir investigando.
Al final, subimos a la plataforma Globaleaks un certificado propio de nuestra organización y funcionó perfectamente.

Un saludo.
________________________________
De: Marcbcn23 ***@***.***>
Enviado: martes, 5 de diciembre de 2023 00:40
Para: globaleaks/GlobaLeaks ***@***.***>
Cc: Rafael Pérez García ***@***.***>; Author ***@***.***>
Asunto: Re: [globaleaks/GlobaLeaks] Nueva instalación de Globaleaks (Discussion #3576)


Hola rpg2023, me encuentro en las mismas. Como lo conseguiste finalmente? No consigo que mi servidor le asigne una dirección que no incluya el puerto de entrada y no consigo ese dichoso certificado. Como fue finalmente? Gracias!
